src/WebJobs.Script/Binding/Http/JsonResolver.cs

```diff
@@ -0,0 +1,29 @@
+using System.Net;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+using System.Reflection;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Script.Binding
+{
+    public class IgnoreErrorPropertiesResolver : DefaultContractResolver
```

Comment:
All of these types should be internal I think. Also, perhaps this can be a private class in the type where it is used?